### ISSUE TITLE
Add LoRa support

### ISSUE THREAD

**Issue by Freyr86** [rgrizzell/LILYGO_T_Deck_CircuitPython#4]
Hello I am actively working to improve the ease of use of the T-deck

I propose here a "nano" module which allows text editing directly on the t-deck
https://github.com/Freyr86/Emu-nano-linux-for-Lilygo-T-deck

And in the second part an emulation of a linux console which allows navigation between files
https://github.com/Freyr86/emul-bash-linux-for-T-Desck

There are two bases to allow the modification of the code in "nomadic" if this interests you

in addition I am going to search your resiprot to modify the code which would allow the use of LoRa I have not yet managed to get it to run with my scripts

If you want to communicate more directly you can find me on Disscord with the same nickname

I would like to thank you for some of your contributions which allowed me to put together my scripts

**Comment by rgrizzell** [comment]
> in addition I am going to search your resiprot to modify the code which would allow the use of LoRa I have not yet managed to get it to run with my scripts

Try using the MicroPython version of the LoRa driver: https://github.com/ehong-tl/micropySX126X
In the future, I will work on getting LoRa to work using a driver written specifically for CircuitPython.

**Comment by Freyr86** [comment]
I am currently testing it but the problem is that it requires a new initialization of the SPI bus and since the screen uses it I cannot use it at the same time. In addition, the library seems to have errors.

If I find a solution I will let you know but it's a bad idea

**Comment by Freyr86** [comment]
Okay, I found the solution, and I'm able to communicate between two T-Decks using LORA. I will post the code soon with a modified library.

**Comment by Freyr86** [comment]
Here is a version of a functional point-to-point chat for the T-Deck if you're interested.

https://github.com/Freyr86/Lilygo-T-deck-LORA

**Comment by rgrizzell** [comment]
> Here is a version of a functional point-to-point chat for the T-Deck if you're interested.
> 
> https://github.com/Freyr86/Lilygo-T-deck-LORA

Thank you! 😄 

I am happy to see that you got that SX126x library working. I am going to create a new CircuitPython driver based on it, and your project will be a helpful working example. My intent is to include that driver in the [CircuitPython Community Bundle](https://github.com/adafruit/CircuitPython_Community_Bundle), much like I've done for this project and the [touchscreen driver](https://github.com/rgrizzell/CircuitPython_GT911).

**Comment by Freyr86** [comment]
Yes, take advantage, on the other hand I made a lot of modifications to the basic one so if I can help don't hesitate

**Comment by Freyr86** [comment]
Update v1.01, I'm going to work on a v2.00 with an additional GUI, along with several tests, it works well.
